src/components/GridLayout/GridLayout.jsx

```diff
@@ -0,0 +1,87 @@
+import { getGridLayoutStyles } from './GridLayout.styles';
+
+import RemoveRedEyeIcon from '@mui/icons-material/RemoveRedEye';
+import DeleteIcon from '@mui/icons-material/Delete';
+
+import ViewPostButton from '../ViewPostButton/ViewPostButton';
+
+import postOne from '../../assets/post1.jpg';
+import postTwo from '../../assets/post2.jpg';
+import postThree from '../../assets/pos3.jpg';
+import postFour from '../../assets/post4.jpg';
+import postFive from '../../assets//post5.jpg';
+import postSix from '../../assets/post6.jpg';
+import { Link } from 'react-router-dom';
+
+const GridLayout = ({ children }) => {
+    const { classes } = getGridLayoutStyles();
+    
+    const userPosts = [
+        {
+            postMedia: postOne,
+            postCaption: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever   since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.",
+            id: 1
+        },
+
+        {
+            postMedia: postTwo,
+            postCaption: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever   since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.",
+            id: 2
+        },
+
+        {
+            postMedia: postThree,
+            postCaption: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever   since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.",
+            id: 3
+        },
+
+        {
+            postMedia: postFour,
+            postCaption: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever   since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.",
+            id: 4
+        },
+
+        {
+            postMedia: postFive,
+            postCaption: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever   since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.",
+            id: 5
+        },
+
+        {
+            postMedia: postSix,
+            postCaption: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever   since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.",
+            id: 6
+        },
+    ];
+
+    return (
+        <Grid container className={classes.gridContainerStyles}>
+            {
+                userPosts.map((gridItem)=>{
```

Comment:
As I can see here this component displays the userPosts, so name the file according.